docs/home.md

```diff
@@ -137,32 +137,32 @@ end)
 Scheduler can be used to register single-use, delayed or repeating tasks.
 ```lua
 -- Schedules task to be run on the next tick.
-scheduler.run(function()
+script.run(function()
```

Comment:
Isn't this meant to be accessing the scheduler table and not the script table? https://github.com/LuaLink/LuaLink/blob/main/src/main/kotlin/xyz/galaxyy/lualink/lua/LuaScriptManager.kt#L29